src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -137,7 +120,7 @@ export const getModuleDetailsUma = async (
       needsBondApproval: needsApproval,
       noTransactions: true,
       activeProposal: false,
-      assertionEvent: {},
+      assertionEvent: undefined,
```

Comment:
used `undefined` here instead of an empty object to make it easier to know when it doesn't exist